prody/utilities/catchall.py

```diff
@@ -968,3 +968,51 @@ def findSubgroups(tree, c, method='naive', **kwargs):
             subgroups[t-1].append(names[i])
 
     return subgroups
+
+
+def findRMSDClusters(rmsd_matrix, c, labels=None):
+    """
+    Divide **rmsd_matrix** into clusters using the gromos method and a cutoff **c**.
+    Returns a list of lists with labels divided into clusters.
+    """    
+    clusters = []
+
+    useful_rmsd_matrix = rmsd_matrix
+    elements = labels
+    
+    indices = list(range(len(elements)))
+    if labels is None:
+        labels = indices
```

Comment:
how can this work if labels is none and indices comes from elements, which comes from labels?